msteams-platform/bots/how-to/Teams conversational AI/conversation-ai-quick-start.md

```diff
@@ -32,27 +33,53 @@ To get started, ensure that you have the following tools:
 
 1. Clone the repository to test the sample app.
 
-   ```bash
+   ```
    git clone https://github.com/microsoft/teams-ai.git
    ```
 
-1. Run the following command to go to the app folder:
```

Comment:
These set up instructions are about to be obsolete. Will follow up with the latest instructions.

Reply:
updated the latest instructions and also provided links to other tools for installation.